src/software/proto/message_translation/ssl_simulation_robot_control.cpp

```diff
@@ -1,14 +1,54 @@
 #include "software/proto/message_translation/ssl_simulation_robot_control.h"
 
+extern "C"
+{
+#include "firmware/shared/physics.h"
+}
+
 std::unique_ptr<SSLSimulationProto::MoveWheelVelocity> createMoveWheelVelocity(
     double front_right, double front_left, double back_left, double back_right)
 {
-    auto move_local_velocity = std::make_unique<SSLSimulationProto::MoveWheelVelocity>();
+    auto move_wheel_velocity = std::make_unique<SSLSimulationProto::MoveWheelVelocity>();
+
+    move_wheel_velocity->set_front_left(static_cast<float>(front_left));
+    move_wheel_velocity->set_front_right(static_cast<float>(front_right));
+    move_wheel_velocity->set_back_left(static_cast<float>(back_left));
+    move_wheel_velocity->set_back_right(static_cast<float>(back_right));
+
+    return move_wheel_velocity;
+}
+
+// Converts rpm and wheel_radius [m] to speed [m/s]
+float rpm_to_m_per_s(float rpm, float wheel_radius)
+{
+    return (2 * (float)M_PI * rpm * wheel_radius) / 60.0f;
+}
+
+std::unique_ptr<SSLSimulationProto::MoveLocalVelocity> createMoveLocalVelocity(
+    double front_right, double front_left, double back_left, double back_right)
```

Comment:
Nit: Can we change `front_right` to something like `front_right_speed`? Upon first reading the code, I thought `front_right` was referring to the wheel itself (just something to denote it's the speed).

Reply:
Fixed